lighthouse-cli/test/smokehouse/lighthouse-runners/bundle.js

```diff
@@ -15,11 +15,11 @@ const ChromeLauncher = require('chrome-launcher');
 const ChromeProtocol = require('../../../../lighthouse-core/gather/connections/cri.js');
 
 // Load bundle, which creates a `global.runBundledLighthouse`.
-// @ts-ignore - file won't exist until `yarn build-all`, but not used for types anyways.
+// @ts-ignore - file exists if `yarn build-all` is run, but not used for types anyways.
```

Comment:
the only remaining `@ts-ignore` since this is sometimes an error and sometimes not, depending on if the local environment has had `yarn build-all` run